crates/ruff_rustpython/src/lib.rs

```diff
@@ -4,9 +4,9 @@ use rustpython_parser::lexer::LexResult;
 use rustpython_parser::{lexer, Mode, ParseError};
 
 /// Collect tokens up to and including the first error.
-pub fn tokenize(contents: &str) -> Vec<LexResult> {
+pub fn tokenize(contents: &str, mode: Mode) -> Vec<LexResult> {
```

Comment:
We could use a `bool` here instead of `Mode` similar to `parse_program_tokens` but I think this is a general function so this is ok.